shell/platform/windows/keyboard_key_embedder_handler.cc

```diff
@@ -264,8 +264,12 @@ void KeyboardKeyEmbedderHandler::KeyboardHookImpl(
     pressingRecords_[physical_key] = eventual_logical_record;
   } else {
     auto record_iter = pressingRecords_.find(physical_key);
-    assert(record_iter != pressingRecords_.end());
-    pressingRecords_.erase(record_iter);
+    if (record_iter == pressingRecords_.end()) {
+      auto critical_iter = critical_keys_.find(key);
+      assert(critical_iter != critical_keys_.end());
```

Comment:
Since the assert will be elided in non-debug builds, continue condensing this to a just a single line:
```cc
assert(critical_keys_.find(key) != critical_keys_.end());
```